embedded/store/immustore.go

```diff
@@ -2321,7 +2323,7 @@ func (s *ImmuStore) FirstTxSince(ts time.Time) (*TxHeader, error) {
 	for left < right {
 		middle := left + (right-left)/2
 
-		header, err := s.ReadTxHeader(middle, false)
+		header, err := s.ReadTxHeader(middle, false, false)
```

Comment:
same comment as above, hard to understand with multiple boolean flags on what is happening

Reply:
yes, even the previous parameter is not clear. Described a possible approach in the previous comment